D/DecompUtil/build_tarballs.jl

```diff
@@ -21,7 +21,7 @@ make install
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms()
+platforms = [p for p in supported_platforms() if triplet(p) != "powerpc64le-linux-gnu"]
```

Comment:
This is a nicer way to exclude the PowerPC architecture:
```suggestion
platforms = supported_platforms(; exclude=p->arch(p)=="powerpc64le")
```

Reply:
cool! updated